tests/Oryx.RuntimeImage.Tests/DotNetCore/LibraryVerificationTest.cs

```diff
@@ -0,0 +1,49 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.RuntimeImage.Tests
+{
+    public class LibraryVerificationTest : TestBase
+    {
+        public LibraryVerificationTest(ITestOutputHelper output) : base(output)
+        {
```

Comment:
Could we add `[Trait("category", "dotnetcore")]` to specify the platform, and probably add a prefix in front of `LibraryVerificationTest` current test name?

Reply:
Maybe change to `DotNetCoreLibraryVerificationTest` in order to match with other tests under dotnetcore platform..